collectors/node/metrics_test.go

```diff
@@ -193,22 +130,22 @@ func TestGetCPUTimes(t *testing.T) {
 	})
 }
 
-func TestGetFilesystems(t *testing.T) {
-	Convey("When getting filesystems", t, func() {
-		Convey("Should return a list containing nodeFilesystem{} structs", func() {
-			fs := getFilesystems()
-			So(len(fs), ShouldBeGreaterThan, 0)
-			So(fs, ShouldHaveSameTypeAs, []nodeFilesystem{})
-		})
-	})
-}
-
-func TestGetNetworkInterfaces(t *testing.T) {
-	Convey("When getting network interfaces", t, func() {
-		Convey("Should return a list containing nodeNetworkInterface{} structs", func() {
-			ifs := getNetworkInterfaces()
-			So(len(ifs), ShouldBeGreaterThan, 0)
-			So(ifs, ShouldHaveSameTypeAs, []nodeNetworkInterface{})
-		})
-	})
-}
+//func TestGetFilesystems(t *testing.T) {
```

Comment:
Cleaned up this commented out code in https://github.com/dcos/dcos-metrics/pull/61/commits/57db8bc140eafe2cdddbcb2ff3bb2fa57c880b60